### PR TITLE
[zh-cn]: update the translation of JavaSrcipt tutorial

### PR DESCRIPTION
**Pull request by ShuoXiTheInari**
### Description

1.  Added the formula of unit conversion（增加了单位的转换公式）
2.  The task description has been corrected to match the actual content（更正了任务描述使其与实际的内容相符）

### Motivation

The description of the task does not match the actual content

### Additional details

None

### Related issues and pull requests

None

### CODE REVIEW

File: files/zh-cn/learn/javascript/first_steps/silly_story_generator/index.md [hunk 1]
```diff
@@ -172,8 +172,8 @@ slug: Learn/JavaScript/First_steps/Silly_story_generator
 4. 在第一个 `if` 块中再次调用这个字符串替换方法，以使 `newStory` 字符串中的名字“李雷”替换为变量 `name` 的值。这里我们说：“如果 `customName` 中有值，就把故事里的“李雷”替换成它。”如果是汉化版将 newStory 中的“李雷”替换成 `name` 的值；
 5. 在第二个 `if` 块中检查 `american` 单选按钮是否被选中。如果选中，就要将故事中的重量和温度值从公斤和摄氏度转换为磅和华氏度，具体事项如下：
 
-   1. 确定英美单位的转换公式。
-   2. 定义变量 `weight`、`temperature` 的行中，分别将美制单位按公式转化为英制，用 `Math.round()` 对计算结果取整。然后将英式单位连接到末尾。
+   1. 确定公制和美制单位的转换公式。
```
Comment (yin1999) [inline files/zh-cn/learn/javascript/first_steps/silly_story_generator/index.md:175]: Please refer the format of [the en-US document](https://github.com/mdn/content/blob/71251662105fbac882aca295c641500175bb9ef1/files/en-us/learn/javascript/first_steps/silly_story_generator/index.md?plain=1#L91)

File: files/zh-cn/learn/javascript/first_steps/silly_story_generator/index.md [hunk 2]
```diff
@@ -187,6 +187,7 @@ slug: Learn/JavaScript/First_steps/Silly_story_generator
   document.querySelector("html").style.backgroundColor = "red";
   ```
 
+- 公制单位与美制单位之间的转换公式 `华氏度 = 32+ 摄氏度× 1.8` `一磅 = 0.45359千克`
```
Comment (yin1999) [inline files/zh-cn/learn/javascript/first_steps/silly_story_generator/index.md:190]: Did not exist in en-US content. Please remove it, as we only do [translations](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md)

**Review by yin1999 — COMMENTED** [review]
Hey @ShuoXiTheInari. Thanks for the contribution.

Because the localized sample code is not currently managed by the MDN teams, and we are still discussing the localization of the sample code. The modifications in the current PR may be no chance of being merged. For now, it's a better idea to fully align the English document.

**Comment by ShuoXiTheInari** [comment]
Thanks for the guidance, I will check the translated instructions again. Since the Chinese version is too different from the English original, maybe I should re-translate the whole document and submit a new PR.

**Pull request closed without merging**